### DISCUSSION TITLE
Client login fails

### DISCUSSION THREAD

**Discussion by llantones** [ltsp/ltsp#188] [Q&A]
Hello:
Sorry about my english.
I have used ltsp 5 before. Now I install ltsp 2019 in Lubuntu 18.04 reading instructions:
https://github.com/ltsp/ltsp/wiki/installation

But the user that login in local in the server, don't login throuth client. Don't shows messages of error. Where can I read logs?

Thanks

**Comment by alkisg** [comment]
There are no "thin clients" in the new LTSP; let's call them "clients" or "LTSP clients".
"Thin" means that the applications run on the server; while in LTSP19, the applications run on the clients.

Try the following steps:

1) Run `sudo ltsp initrd`, then reboot the client and see if it can login. If this works, remember, you need to run `ltsp initrd` every time you add a new user or you modify ltsp.conf.

2) If (1) doesn't work, try this: on the client, press Ctrl+Alt+F2 to switch to virtual console 2, and try to login in that terminal, with the username and password. Do you see any error messages while doing that?

3) You may also try `ssh user@server` from another PC in your network, to verify that the user and the SSH server are functioning properly.

**Comment by llantones** [comment]
I can login in the console.

**Comment by alkisg** [comment]
How much RAM does the client have? Is it enough to run a graphical environment?

One more troubleshooting step would be to login to the client console as administrator (a sudoer). Then run:

```shell
sudo nano /usr/share/xsessions/*.desktop
```

Find a line there that says:
Exec=lubuntu-session

...or something similar. Change it to:
Exec=lxterm

Save, and exit. Then, when you try to login again, you'll get a terminal instead of a graphical environment. From that terminal, try to run `lubuntu-session` or whatever the initial Exec command was, and see the errors in the terminal output.

**Comment by llantones** [comment]
Intel atom with 1 GB.

I change to Exec=lxterm and reboot the client but it start lxsession instead lxtrem.
Thanks

**Comment by alkisg** [comment]
Don't reboot after changing that line, because then the change is lost.
Just try to login.

**Comment by llantones** [comment]
If I reboot server, after changed *.desktop, I login in lxterm, but the client don't. 

If I made that you said to me, change and not reboot, not login in lxterm.

**Comment by alkisg** [comment]
I was asking to change the client.

If you changed the server, you need to run `ltsp image /`, for the change to go to the client.

**Comment by llantones** [comment]
Sorry, I always thinking in ltsp5 and thin clients...

Now I did in the client:
> One more troubleshooting step would be to login to the client console as administrator (a sudoer). Then run:

> sudo nano /usr/share/xsessions/*.desktop

> Find a line there that says:
> Exec=lubuntu-session

> ...or something similar. Change it to:
> Exec=lxterm

The exact line is:
Exec=/usr/bin/lxsession -s Lubuntu -e LXDE
and I wrote:
`Exec=lxterm`
and
`Exec=/usr/bin/lxterm`
but it doesn't go.

**Comment by alkisg** [comment]
If you can login to console, but you can't login to lxterm, something very weird is going on.

Try to come online in IRC for remote support.

**Comment by llantones** [comment]
I never use IRC before, you say me...

**Comment by alkisg** [comment]
Just go to https://ltsp.github.io/ and click on the link that says IRC channel.

**Comment by llantones** [comment]
[SOLVED]
The reason is that lxde fails, so the far solution: use ligthdm.
I think the community will sove the error in lxde.
Thanks

**Comment by alkisg** [comment]
@llantones, why was lxdm installed? I don't see it in the live CD of Lubuntu 18.04, the one I initially tested with.

If you want, you may click the "Close and comment" button, so that the issue is marked as closed.

**Comment by llantones** [comment]
> why was lxdm installed?

I don't remember, maybe lubuntu netinstall instead full cd?
or that I prefer lxdme because in lightdm it's necesary usr the mouse to select the user.
